packages/google_maps_flutter/google_maps_flutter_android/README.md

```diff
@@ -0,0 +1,12 @@
+# google\_maps\_flutter\_android
+
+The Android implementation of [`google_maps_flutter`][1].
+
+## Usage
+
+This package is [endorsed][2], which means you can simply use
```

Comment:
Link to README of `google_maps_flutter` would be useful here to smooth out the user journey

Reply:
It's already linked a few lines above. (Also, the standard user journey is "nobody ever sees this file, or thinks much about this package at all" 🙂 )